nomad-base/CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 ### Unreleased
 
+- emit event for source of config loaded at bootup
```

Comment:
Nit: I would clarify this since we've changed more than just adding logging for config loading